app/assets/javascripts/base.js.erb

```diff
@@ -236,9 +238,6 @@ $(function() {
     var selected_images = container.find('.selected:not(.group.selected .selected)').children('.image');
     selected_images.clone().appendTo(selected_container);
     /* each of the images needs to be pointing to the smaller image. boom. */
-    selected_container.find('img').each(function(){
```

Comment:
I think this is from an older version of the UI. It causes not found errors anyway, since these images have different asset digest hashes, so just switching the dimension part of the path doesn't do anything. I removed this and tested all search types and didn't notice any problems.